### ISSUE TITLE
IRSM board not binding with ACCST receivers

### ISSUE THREAD

**Issue by DeeBee878** [opentx/opentx#7442]
I ugraded my Horus X10s RF board to the ISRM board.  I purchassed the upgrade kit from Horus RC who have been helpful and advised me to install back in November 2019 that I needed to instal OpenTx 2.3.3 followed by intalling the ISRM firmware found on the FrSky website under X10 Exspress.
It worked and I was able to bind ACCST and I test flew successfully in small foamie.  When I upgrades receivers to ACCST V2 as advised by FrSky those receivers would no longer bind to my X10s.
I upgraded 2 receivers to ACCESS firmware and they did bind and worked reliably.
I could not set the antenna to full time external however.  If I did, the external antenna would work but only until I switched  the Tx off.  Upon restart, it would give a defective antenna warning.

I have now installed OpenTx 2.3.6 and reinstalled ISRM V1.1.3 firmware as found on the FrSky website.  It binds well to my two ACCESS receivers but does not bind ACCST receivers either orignal V1 or the recent V2 firmware.
Range testing is good with extern antenna outperforming the internal by 3 to 4 RSSI at approx. 60m with TX  set on Range testing mode.
I no longer get a defective antenna warning although twice i noticed a message saying 'module upgrade required' has appeared when on an ACCST model setting.  Not sure if it will do that in ACCESS.
In summary, with 2.3.6, no ACCST D16 binding and ocassional message saying module upgrade required.  I am  not sure if it is an FrSky firmware issue or something that can be addressed in OpenTX.

**Comment by kilrah** [comment]
1.1.3 for the upgrade ISRM is NOT SAFE and must not be used. That's why you get the "module upgrade required" message.
It is not distributed on the FrSky website anymore for a reason. If you found one you went to the wrong product page and it is not the right firmware for your module, only what's on the "ACCESS Upgrade Module Kit (X10 & X10S)" page is.

https://www.frsky-rc.com/access-upgrade-module-kit-x10-x10s/

**Comment by DeeBee878** [comment]
Thank you Kilrah for correcting me.
I did not realise what I had done.
I have installed ISRM 1.1.5 and I now have both internal and external antennas.
I can now bind ACCST V1 receivers successfully.
I still cannot bind ACCST receivers that have been updated to V2 ACCST.
Is this something only FrSky can address and is it safe to test fly on my current set up with OpenTx 2.3.6, or should I be waiting for further updates?
Dennis

**Comment by 3djc** [comment]
Yes, only FrSky can address binding accst v2 from ISRM

**Comment by kilrah** [comment]
That is normal, V2 ACCST will be a new update from FrSky that's yet to come. 
You should be using OpenTX 2.3.5, not 2.3.6 so redownload and reflash.

As for safety of ACCST v1 and ACCESS (which needs an update too) there are many differing opinions on the net, it's all outside of our control and expertise so won't give advice.

**Comment by DeeBee878** [comment]
Thanks.  Done that and it seems to be working OK apart ACCST V2 of course
